File: java2csharp/java/Samsung/Camera/DisabledSettingItem.java
package com.samsung.android.sdk.camera.sample.cases.util;

import android.app.Activity;
import android.support.v4.content.ContextCompat;
import android.view.View;
import android.widget.TextView;

import com.samsung.android.sdk.camera.sample.R;

/**
 * Dummy Setting Item for setting feature that is not supported by the device.
 * This item only show the gary outed text label.
 */
public class DisabledSettingItem extends SettingItem<Object, Object>{
    public DisabledSettingItem(Activity activity, String name) {
        super(activity, name, null);
        mView = mActivity.getLayoutInflater().inflate(R.layout.setting_list_item, null);
        ((TextView)mView.findViewById(R.id.title)).setText(name);
        ((TextView)mView.findViewById(R.id.title)).setTextColor(ContextCompat.getColor(mActivity, android.R.color.secondary_text_dark));
        mView.findViewById(R.id.itemlist).setVisibility(View.GONE);
    }

    @Override
    public Object getValue() {
        return null;
    }

    @Override
    public void setValue(Object value) { }

    @Override
    public void setEnabled(boolean enable) { }

    @Override
    public boolean isEnabled() {
        return false;
    }
}

File: java2csharp/csharp/Samsung/Camera/cases/util/DisabledSettingItem.cs
namespace com.samsung.android.sdk.camera.sample.cases.util
{

	using Activity = android.app.Activity;
	using ContextCompat = android.support.v4.content.ContextCompat;
	using View = android.view.View;
	using TextView = android.widget.TextView;

	/// <summary>
	/// Dummy Setting Item for setting feature that is not supported by the device.
	/// This item only show the gary outed text label.
	/// </summary>
	public class DisabledSettingItem : SettingItem<object, object>
	{
		public DisabledSettingItem(Activity activity, string name) : base(activity, name, null)
		{
			mView = mActivity.LayoutInflater.inflate(R.layout.setting_list_item, null);
			((TextView)mView.findViewById(R.id.title)).Text = name;
			((TextView)mView.findViewById(R.id.title)).TextColor = ContextCompat.getColor(mActivity, android.R.color.secondary_text_dark);
			mView.findViewById(R.id.itemlist).Visibility = View.GONE;
		}

		public override object Value
		{
			get
			{
				return null;
			}
			set
			{
			}
		}

		public override bool Enabled
		{
			set
			{
			}
			get
			{
				return false;
			}
		}
	}

}